src/server/qgsservercachefilter.cpp

```diff
@@ -0,0 +1,89 @@
+/***************************************************************************
+                          qgsservercachefilter.cpp
+                          ------------------------
+ Cache interface for Qgis Server plugins
+
+  begin                : 2018-07-05
+  copyright            : (C) 2018 by René-Luc D'Hont
+  email                : rldhont at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsservercachefilter.h"
+
+#include <QDomDocument>
+
+QgsServerCacheFilter::QgsServerCacheFilter( const QgsServerInterface *serverInterface )
+  : mServerInterface( serverInterface )
+{
+}
+
+QByteArray QgsServerCacheFilter::getCachedDocument( const QgsProject *project, const QgsServerRequest &request, const QString &key ) const
+{
+  Q_UNUSED( project );
+  Q_UNUSED( request );
+  Q_UNUSED( key );
```

Comment:
Generally, I prefer not to give a name to a variable instead of using `Q_UNUSED( ... )`.

In this case:

```
QByteArray QgsServerCacheFilter::getCachedDocument( const QgsProject *, const QgsServerRequest &, const QString & ) const
```

But it's a minor detail :).